languages/scala/reference/exercise-concepts/raindrops.md

```diff
@@ -0,0 +1,42 @@
+# Rain drops
+
+[Example implementation](https://github.com/exercism/scala/tree/master/exercises/raindrops)
+[Other implementation](https://scastie.scala-lang.org/JixK2hdPQ4ier47Wj9aM9A):
+```
+object Raindrops {
+  private val sounds = Map(
+    3 -> "Pling",
+    5 -> "Plang",
+    7 -> "Plong"
+  )
+
+  def convert(n: Int): String =
+    sounds
+      .filterKeys(n % _ == 0)
+      .values match {
+      case s if s.isEmpty => n.toString
+      case s              => s.mkString
+    }
+}
+```
+
+## General
+
+- object or case class: used as class wrapper for exercise
+- functions: used as main entry point for the exercise
+- function arguments: input strands passed as arguments
+- return values: return a value from a function
```

Comment:
I think the "integers," "strings," and "booleans" Concepts should also be added.